ionoscloud/resource_server.go

```diff
@@ -1356,6 +1356,10 @@ func setResourceServerData(ctx context.Context, client *ionoscloud.APIClient, d
 				nicId = *nic.Id
```

Comment:
@cristiGuranIonos looking at this again I wonder if we can just get rid of this check, because, basically, in this check, we are setting the `nicId` as being the `ID` of the last `NIC` in the list of `NICs` that has `LAN` 1. So instead of doing this, I think we can do:
```
nicId = *(*server.Entities.Nics.Items)[0].Id
```
directly since we already know that the list contains at least one `NIC` and the `ID` of the first `NIC` is not `nil`.

More than that, we decided that we should populate this with the FIRST `NIC` from the list, but with this code I'm not sure if this happens all the time, maybe there are some cases in which the `NIC` that is in the first `LAN` is not the first in the list. What do you think?

Reply:
I think we should just remove the for that checks for lan 1 and just set the first id we find in the list